frontend/modules/recipe/components/Recipe.js

```diff
@@ -172,7 +193,7 @@ var RecipeScheme = injectIntl(React.createClass({
                 { (this.props.data.source) ?
                   <div>{ formatMessage(messages.source) }: { this.props.data.source }</div>
                   :
-                  <div>{ formatMessage(messages.created_by) }: { this.props.data.username }</div>
+                  <div>{ formatMessage(messages.created_by) }: { this.props.data.username } { this.showEditLink() }</div>
```

Comment:
When I added the source field, I wanted it as a way of tracking a recipe that was copied from some other site. The only reason that I didn't show both was I couldn't find a way to make it look nice. Feel free to play around with the styles if you wanna display both of them.